keel-core/src/test/kotlin/com/netflix/spinnaker/keel/constraints/ManualJudgementConstraintEvaluatorTests.kt

```diff
@@ -65,6 +67,8 @@ internal class ManualJudgementConstraintEvaluatorTests : JUnit5Minutests {
   fun tests() = rootContext<Fixture> {
     fixture { Fixture }
 
+    before { clearAllMocks() }
```

Comment:
![image](https://user-images.githubusercontent.com/95210/111699464-653a7800-87f5-11eb-9568-2313697478a9.png)

Reply:
@robfletcher I thought of you when i typed this but it legit did not work before I did it and after I added it it worked. So what am I missing here????

Reply:
I added suggestions below. It's because `Fixture` is an object rather than a class so all the tests end up using the same instance. Probably my fault.

Reply:
oh that's SOOOO great to finally understand.